wordpress-mobile/WordPress-iOS
# Add Blogging Reminders "Get Started" card

Fixes #16527 

Known issues:
- The buttons do nothing at this moment

To test:

- Build run and make sure the blogging reminders card appears and looks consistent with the designs (see screenshot below)
-  Make sure the My Site screen behaves as expected (can navigate to subsection, quick actions work etc)
- Disable the `bloggingReminders` feature flag, make sure the blogging reminders card does not show up, and also make sure that My Site behaves as expected
- Test on iPhone and iPad

<p align=center>
<img height=450 src=https://user-images.githubusercontent.com/34376330/118903918-5bf58680-b8de-11eb-840f-bc5d0a951195.png>
</p>

## Regression Notes
1. Potential unintended areas of impact
Low, a feature flagged section was added to the tableview in `BlogDetailsViewController`

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Smoke test on the features of my site


3. What automated tests I added (or what prevented me from doing so)
None, these are purely UI changes
PR submission checklist:
